vscode-trace-common/src/messages/vscode-message-manager.ts

```diff
@@ -209,4 +210,9 @@ export class VsCodeMessageManager extends Messages.MessageManager {
         const data = JSON.stringify(payload);
         vscode.postMessage({ command: VSCODE_MESSAGES.CONTEXT_MENU_ITEM_CLICKED, data: data });
     }
+
+    sourceLookup(path: string, line: number): void {
+        const data = { path: path, line: line };
```

Comment:
I troubleshooting a use case with the Trace Compass server and I noticed that the events table data provider for certain trace types will return a list of source in the form [file1:line1, file2:line2]. Even if it's only one file it will provide it with the brackets. I think when we did this prototype, there was only never the brackets. Anyways, I think for now it's ok to not support the list. I'll need to think about more how to support that better.

Is your format always `file:line` or is it also [file:line]`, i.e. with brackets?

Reply:
Thanks, I think I've addressed the immediately actionable review comments.   The fact that callstacks can carry source annotations (in a list form) is new to me.  Going forward, I'll try to learn more about that specific case (how the data flow works for that), but for now, our only usage of "Source" attribute is as an aspect of our ITmfEvent subclass, and it's only file:line.